filescan.go

```diff
@@ -111,8 +115,27 @@ func (s *FileScanner) Scan(r io.Reader, filename string, progress chan<- float32
 	return analysis, nil
 }
 
+// Scan sends a file to VirusTotal for scanning.
+func (s *FileScanner) Scan(r io.Reader, filename string, progress chan<- float32) (*Object, error) {
+	return s.VtFileUpload(r, filename, "files", nil, progress)
+}
+
 // ScanFile sends a file to VirusTotal for scanning. This function is similar to
 // Scan but it receive an *os.File instead of a io.Reader and a file name.
 func (s *FileScanner) ScanFile(f *os.File, progress chan<- float32) (*Object, error) {
 	return s.Scan(f, f.Name(), progress)
 }
+
+// MonitorUploadFilename uploads a file to your VT Monitor account using a destination path
+func (s *FileScanner) MonitorUploadFilename(r io.Reader, monitorPath string, progress chan<- float32) (*Object, error) {
```

Comment:
I don't think that `FileScanner` should have a methods related to Monitor. That's confusing because the `FileScanner` object is supposed to be used for scanning files with VirusTotal, and Monitor is a completely unrelated feature. 

We could have a `MonitorUploader` class that implements `Upload(r io.Reader, filename string, progress chan<- float32)` and `UploadFile(f *os.File, progress chan<- float32)`.

Reply:
resolved as recommended in 2f5296f11f72fd72a9b6539c16606d40e2fd6f1d